CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ ENHANCEMENTS:
 
 BUG FIXES:
 * Added a check in the `tofu test` to validate that the names of test run blocks do not contain spaces. ([#1489](https://github.com/opentofu/opentofu/pull/1489))
+* Fix for `tofu test` when the module has no resources, its outputs become unusable. ([#1409](https://github.com/opentofu/opentofu/pull/1409))
```

Comment:
Can you talk about the positive effect of this now? maybe saying something like...

`tofu test` now supports accessing module outputs when the module has no resources.

what do you think? I think we should always try and focus on the positive (the fix) and not the negative (the bug)

Reply:
sure, that makes more sense.